apex-squid/src/main/java/org/fundacionjala/enforce/sonarqube/apex/api/ApexGrammar.java

```diff
@@ -28,23 +29,26 @@ private ApexGrammar() {
      * It is the main method of grammar. Here all other grammars are
      * constructed.
      *
+     * @param errorRecoveryEnabled error recovery status
      * @return the grammar
      */
-    public static Grammar create() {
+    public static Grammar create(boolean errorRecoveryEnabled) {
```

Comment:
Right, I just took that part from the other plugin implementations, since more than one of them used it, but it won't be hard to change it. Should I remove it, then? @ziur @JimmyRojas